ui/src/dashboards/components/createFromTemplateOverlay/DashboardTemplateDetails.tsx

```diff
@@ -0,0 +1,149 @@
+// Libraries
+import React, {PureComponent} from 'react'
+import _ from 'lodash'
+
+// Components
+import {
+  Panel,
+  EmptyState,
+  ComponentSize,
+  Grid,
+  Columns,
+  DapperScrollbars,
+} from '@influxdata/clockface'
+import {TemplateSummary, ITemplate} from '@influxdata/influx'
```

Comment:
I think we should prefer to import from `src/types` instead of directly from client since sometimes types are altered in the process.